common/raw_hashtable.h

```diff
@@ -1524,6 +1607,19 @@ auto TableImpl<InputBaseT, SmallSize>::small_storage() const -> Storage* {
   }
 }
 
+// Helper to setup the storage of a table when a specific size has already been
+// setup. If possible, uses any small storage, otherwise allocates.
```

Comment:
```suggestion
// Helper to set up the storage of a table when a specific size has already been
// set up. If possible, uses any small storage, otherwise allocates.
```

Reply:
("setup" is a noun; the verb is two words.) I'd also be inclined to rename the function -> `SetUpStorage.`